src/splunk_otel/distro.py

```diff
@@ -32,3 +33,18 @@ def _configure(self, **kwargs):
     def set_env_defaults(self):
         for key, value in DEFAULTS.items():
             self.env.setdefault(key, value)
+
+    def load_instrumentor(self, entry_point, **kwargs):
+        #  This method is called in a loop by opentelemetry-instrumentation
+        if is_system_metrics_instrumentor(entry_point) and not self.env.is_true(OTEL_METRICS_ENABLED):
+            print(f"{OTEL_METRICS_ENABLED} not set -- skipping SystemMetricsInstrumentor")
```

Comment:
Shouldn't this be a `logger.` or something other than a bare `print`?

Reply:
Fixed.